packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/upgrade-storage/upgrade-storage.html

```diff
@@ -94,9 +104,26 @@
                                 >
                                 </ovh-manager-catalog-price>
                                 <span
-                                    data-translate="pci_databases_general_information_upgrade_storage_summary_price_hourly_unit"
+                                    data-ng-bind="($ctrl.showMonthlyPrices ? 'pci_databases_general_information_upgrade_storage_summary_price_monthly_unit' : 'pci_databases_general_information_upgrade_storage_summary_price_hourly_unit') | translate"
+                                ></span>
+                            </div>
+                            <div
+                                class="oui-color-aw-500 bold"
+                                data-ng-if="$ctrl.additionalDiskSize - $ctrl.originallyAddedDiskSize < 0"
+                            >
+                                <ovh-manager-catalog-price
+                                    data-price="$ctrl.priceDelta"
+                                    data-tax="$ctrl.taxDelta"
+                                    data-user="$ctrl.user"
+                                    data-show-zero-price="true"
+                                    data-perform-rounding="false"
+                                    data-maximum-fraction-digits="3"
+                                >
+                                </ovh-manager-catalog-price>
+                                <span
+                                    data-ng-bind="($ctrl.showMonthlyPrices ? 'pci_databases_general_information_upgrade_storage_summary_price_monthly_unit' : 'pci_databases_general_information_upgrade_storage_summary_price_hourly_unit') | translate"
```

Comment:
you have the same code L.107. You should create a function in the controller to use it in both places

Reply:
Changed here: https://github.com/ovh/manager/pull/9612/commits/63ebc930716d62488f1284e0bdb832771af1b6ed